magento/magento2
# Add mass action to invalidate indexes via admin

### Description (*)
This pull request adds the ability for an administrator to mark indexes as invalid via the admin. The indexes will then be run by the normal minutely cron.

### Fixed Issues (if relevant)
I am not aware of any open issues regarding this change.

### Manual testing scenarios (*)
1. Browse to admin -> System -> Tools -> Index Management
1. Select index tickbox
1. Use mass-action drop-down to select 'invalidate index'
1. Click 'submit'
1. Observe index is now marked as invalid

### Questions or comments
I have used the existing permission "Change indexer mode", but can move this to its own permission category if desired.

I copied the 'masschangelog' controller & test to make this action.

### Contribution checklist (*)
